app/src/main/java/org/stepik/android/remote/course_list/service/CourseListService.kt

```diff
@@ -0,0 +1,11 @@
+package org.stepik.android.remote.course_list.service
+
+import io.reactivex.Single
+import org.stepik.android.remote.course_list.model.CourseCollectionsResponse
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface CourseListService {
```

Comment:
naming and features should be normalised
now we have:
- course-lists - api endpoint
- course-list feature name
- course collection model name (?)